platform/platform-scheduler/src/main/java/ddf/platform/scheduler/ScheduledCommandTask.java

```diff
@@ -34,17 +37,26 @@
  *
  * @author Ashraf Barakat
  * @author ddf.isgs@lmco.com
- *
  */
 public class ScheduledCommandTask implements ScheduledTask {
 
+    public static final String SECOND_INTERVAL = "secondInterval";
+
+    public static final String CRON_STRING = "cronString";
```

Comment:
It feels like the interval type strings should be an enum. It's not a major problem, just my preference.